src/main/java/uk/gov/hmcts/darts/common/entity/CaseRetentionEntity.java

```diff
@@ -38,13 +38,32 @@ public class CaseRetentionEntity extends CreatedModifiedBaseEntity {
     private CourtCaseEntity courtCase;
 
     @ManyToOne
```

Comment:
Please can you check these ManyToOne relationships be similar to your change in AnnotationEntity?
```
    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = ..., nullable = false)
```
Only columns cmr_id, comments, retention_object_id look to nullable in the `retention.sql`